modules/aioseop_sitemap.php

```diff
@@ -3419,6 +3740,25 @@ function get_tax_args( $page = 0 ) {
 				$args['offset'] = $page * $this->max_posts;
 			}
 
+			/**
+			 * Filter to add additional taxonomy query for searching items to add to the sitemap.
+			 *
+			 * @since ?
+			 *
+			 * @param array $args {
```

Comment:
Could probably just reference WP's documentation. The aioseop options could be referenced if there was documentation; which would be for a different time.

```
/**
 * Filter to add additional taxonomy query for searching items to add to the sitemap.
 *
 * @since ?
 *
 * @param array $args {
 *     Arguments to send to WP_Query.
 *
 *     @see Parameter $args for WP_Query.
 *     @link https://codex.wordpress.org/Class_Reference/WP_Query#Parameters
 * }
 * @param int       $hardcoded      Page number.
 * @param array $options {
 *      Array of the module's options.
 *
 *      TODO This needs a reference, and reference also needs to be documented. Options is ~98 variables/keys.
 * }
 */
```